axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,201 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+class Tranquilizer(Player):
+   
+    """
+    Submitted to Axelrod's second tournament by Craig Feathers
+
+    This strategy is based on the reverse engineering of the 
+    Fortran strategy K67R from Axelrod's second tournament.
+    Reversed engineered by: Owen Campbell, Will Guo and Mansour Hakem.
+
+
+    Description given in Axelrod's "More Effective Choice in the 
+    Prisoner's Dilemma" paper: The rule normally cooperates but 
+    is ready to defect if the other player defects too often. 
+    Thus the rule tends to cooperate for the first dozen or two moves
+     if the other player is cooperating, but then it throws in a 
```

Comment:
The docs are still breaking, sphinx is super sensitive with blank space :)

Here you need to remove the space before the `if`.